STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -903,10 +920,7 @@ Internet (MAMI). This support does not imply endorsement.
 # CSR Template Schema
 {: #csr-template-schema}
 
-Following is a JSON Schema definition of the CSR template. The syntax used is
-that of draft 7 of JSON Schema, which may not be the latest version of the
-corresponding Internet Draft {{!I-D.handrews-json-schema}} at the time of
-publication.
+Following is a JSON Schema definition of the CSR template. The syntax used is that of draft 7 of JSON Schema, which is documented in {{json-schema-07}}. Note that later versions of this (now expired) draft describe later versions of the JSON Schema syntax. At the time of writing, a stable reference for this syntax is not available yet.
```

Comment:
We should say here that this is a conscious decision to stay on draft-7 rather than updating to more recent releases because that is what has got widespread implementation.

Reply:
At the time of writing, a stable reference for this syntax is not yet available, and we have chosen to use the draft version which is currently best supported by tool implementations.